flows/interfaces.go

```diff
@@ -224,6 +210,7 @@ type Timeout interface {
 // Wait tells the engine that the session requires input from the user
 type Wait interface {
 	utils.Typed
+	FlowTypeRestricted
```

Comment:
A theoretical wait type is the "nothing" wait type - used to get the session committed before moving on to something else. In theory it could be part of a passive flow. Anyways cross that bridge if we ever get to it!